### PR TITLE
Various CCPP PRs: change vertical dimension and long name of dku/dkt, UGWP bugfixes, LTP bugfixes, sfcsub update, framework update

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

This PR:
- contains #288, ugwpv1_gsldrag bug fixes, and GFS_typdefs updates associated with diag flags for ugwpv1_gsldrag.F90 and GFS_DCNV_generic.F90)
- changes the standard name and vertical dimension for arrays `dkt` and `dku` 
- updates submodule pointers for ccpp-framework and ccpp-physics for the changes in the associated PRs listed below.

### Issue(s) addressed

See https://github.com/NCAR/ccpp-physics/pull/641.

## Testing

For regression testing, see https://github.com/ufs-community/ufs-weather-model/pull/541

The changes in https://github.com/NCAR/ccpp-physics/pull/635/files modify the answers of all regression tests involving Thompson MP.

The changes in https://github.com/NCAR/ccpp-physics/pull/634/files modify the answers of all regression tests involving UGWP v1.

## Dependencies

https://github.com/NCAR/ccpp-framework/pull/368
https://github.com/NCAR/ccpp-physics/pull/641
https://github.com/NOAA-EMC/fv3atm/pull/291
https://github.com/ufs-community/ufs-weather-model/pull/541

### CODE REVIEW

File: ccpp/data/GFS_typedefs.meta [hunk 1]
```diff
@@ -6940,16 +6940,16 @@
...
+  long_name = atmospheric heat diffusivity
   units = m2 s-1
-  dimensions = (horizontal_loop_extent,vertical_dimension_minus_one)
+  dimensions = (horizontal_loop_extent,vertical_dimension)
   type = real
   kind = kind_phys
 [dku]
   standard_name = atmosphere_momentum_diffusivity
-  long_name = diffusivity for momentum
+  long_name = atmospheric momentum diffusivity
```
Comment (junwang-noaa) [inline ccpp/data/GFS_typedefs.meta:6950]: @climbfuji Would you please also change the descriptions of dkt and dku with the same long name in ccpp/driver/GFS_diagnostics.F90?

Comment (climbfuji) [reply ccpp/data/GFS_typedefs.meta:6950]: Good point, done.

**Review by junwang-noaa — APPROVED** [review]

**Review by BinLiu-NOAA — APPROVED** [review]
Changes look good to me.

**Comment by climbfuji** [comment]
This PR is ready to merge.

**Pull request merged**